src/test/scala/za/co/absa/commons/config/ConfigurationImplicitsSpec.scala

```diff
@@ -225,4 +225,59 @@ class ConfigurationImplicitsSpec extends AnyFlatSpec with Matchers {
       conf getOptionalString "oops" should be(None)
     }
   }
+
+  behavior of "ConfigurationMapWrapper"
+
+  it should "implement convert config to map of Strings" in {
+    val configuration = new MapConfiguration(Map("foo" -> "1", "bar" -> "2").asJava)
+    val map = configuration.toMap[String]
+    map.apply("foo") should be("1")
+    map.apply("bar") should be("2")
+  }
+
+  it should "implement convert config to map of Integers" in {
+    val configuration = new MapConfiguration(Map("foo" -> "1", "bar" -> 2).asJava)
+    val map = configuration.toMap[Int]
+    map.apply("foo") should be(1)
+    map.apply("bar") should be(2)
+  }
+
+  it should "implement convert config to map of Longs" in {
+    val configuration = new MapConfiguration(Map("foo" -> "1", "bar" -> 2).asJava)
+    val map = configuration.toMap[Int]
```

Comment:
I would expect type Long in this test.
```suggestion
    val map = configuration.toMap[Long]
```

Reply:
Unit tests executed.

Reply:
Sure, This is only note that related unit tests have been executed.

Reply:
ok, do you see anything else that needs to be fixed? If not please approve the PR.